src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using Microsoft.PowerFx.Core.Types.Enums;
+
+namespace Microsoft.PowerFx.Core.Public.Config
+{
+    /// <summary>
+    /// A container that allows for compiler customization.
+    /// </summary>
+    public sealed class PowerFxConfig
+    {
+        private readonly EnumStore _enumStore;
```

Comment:
>EnumStore [](http://example.com/codeflow?start=25&length=9)

Another source of "intellisense noise" is colors: 
\Power-Fx\src\libraries\Microsoft.PowerFx.Core\Types\Enums\ColorTable.cs

Not sure what to do there. Maybe those are common enough we always include. Maybe hosts need more control.

Reply:
Most of the contents of the enum store here should be removed and moved back to PA-Client/other hosts as needed

Reply:
I will be removing static data in future PRs. I am going to be splitting them up into smaller changes to make it easier to integrate with CanvasApps.